### PR TITLE
Add created rent email

### PR DESCRIPTION
**Pull request by Martum**
* Add sidekiq and dotenv gems
* Add timestamps to Rents

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -13,13 +13,34 @@
 development:
   secret_key_base: 212c3d67c1c1017b8a6497f7ab7a5837a725a1d444cf95e96bb6ff9a9f9acc719f62f4558aecd7dd6b2cf0f3167c2def3d75305dd6de9e0fb75bb290e2894ab4
   devise_sender_email: "no-reply@wbooksapi.com.ar"
+  redis_url: "redis://localhost:6379/0"
+  email:
+    port: 587
```
Comment (francoprud) [inline config/secrets.yml]: I never saw this identation inside `secrets.yml`..nice!

File: config/environments/production.rb [hunk 2]
```diff
@@ -61,6 +61,16 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.smtp_settings = {
+    authentication: :plain,
+    address: Rails.application.secrets.email["address"],
```
Comment (francoprud) [inline config/environments/production.rb]: Why not calling `..secrets.email.address`?

Same below

File: config/sidekiq.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+:queues:
+  - default
```
Comment (francoprud) [inline config/sidekiq.yml]: MIssing `mailers` queue?

**Comment by francoprud** [comment]
@Martum Also add the routes for sidekiq web dashboard!

File: config/environments/production.rb (hunk 2 above)
Comment (Martum) [inline config/environments/production.rb]: Rails creates a hash with the extra-indented keys, they are only available with `[]`.

**Pull request merged**